src/components/markers/pokemon.jsx

```diff
@@ -31,6 +31,7 @@ export const fancyMarker = (
   pkmn,
   glow,
   ivCircle,
+  levelCircle,
```

Comment:
Please add this new arg as the last one for the function, to help avoid breaking any custom marker files people may have.